CONTRIBUTING.md

```diff
@@ -10,33 +10,36 @@ For general information about the project, see [our readme](https://github.com/z
 Most discussions happen on our [Discord Server](https://discord.zelda64.dev) where you are welcome to ask if you need help getting started, or if you have any questions regarding this project and other decompilation projects.
 
 
-Useful Links
-------------
+## Useful Links
 
-- [Discord](https://discord.zelda64.dev/) - Primary discussion platform.
-- [Google Sheets reservation board](https://docs.google.com/spreadsheets/d/1X83YCPRa532v-Zo0WgUsJ2kB1X9RxBta5_p9aWA8uro) - We use this to track decompilation progress, not GitHub Issues.
+- [Code Files Google Sheet](https://docs.google.com/spreadsheets/d/1X83YCPRa532v-Zo0WgUsJ2kB1X9RxBta5_p9aWA8uro) - We use this to track decompilation progress and file reservations, instead of GitHub Issues or Trello or similar.
 
 - [Installation guide](https://github.com/zeldaret/mm/blob/master/README.md#installation) - Instructions for getting this repository set up and built on your machine.
+- [Style Guide](docs/STYLE.md) - Description of the project style that we ask contributors to adhere to.
 - [Code Review Guidelines](REVIEWING.md) - These are the guidelines that reviewers will be using when reviewing your code. Good to be familiar with these before submitting your code.
```

Comment:
Since `STYLE.md` got put in `docs`, I feel like `REVIEWING.md` should also go there. Perhaps `CONTRIBUTING.md` as well, but that one I'm not as sure on.

Reply:
Good question, would welcome others' thoughts. I think CONTRIBUTING.md has to be in root for github to recommend it?

Reply:
Seems like `CONTRIBUTING.md` can go in the root, `docs`, or `.github` folders. https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors.

Reply:
Ah, in that case I'm happy to put them all in the docs folder.